feature-libs/order/components/return-request-list/order-return-request-list.component.spec.ts

```diff
@@ -80,14 +90,35 @@ describe('OrderReturnRequestListComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  it('should not set sort type if no return request are provided', () => {
+    let returns: ReturnRequestList | undefined;
+
+    mockReturnRequestList$.next({});
+
+    component.returnRequests$
+      .subscribe((value) => {
+        returns = value;
+      })
+      .unsubscribe();
+
+    expect(returns).toEqual({});
+    expect(component.sortType).toBe(undefined);
+  });
+
```

Comment:
are changes in this file related to this PR?

Reply:
Effort done by @mszczepaniak to bump unit test coverage above the defined threshold for pipeline